inference_perf/config.py

```
@@ -48,8 +49,21 @@ class LoadConfig(BaseModel):
     stages: List[LoadStage]
 
 
+class ReportStorageConfig(BaseModel):
+    google_cloud_storage: Optional[GoogleCloudStorageConfig] = None
+
+    @model_validator(mode="after")
+    def check_oneof(self) -> "ReportStorageConfig":
+        storage_backends = [
+            self.google_cloud_storage,
+        ]
+        if sum(v is not None for v in storage_backends) > 1:
+            raise ValueError("No more than one storage backend may be set in ReportStorageConfig.")
+        return self
+
+
 class ReportConfig(BaseModel):
```

can we add an enum type for the report - defaulting to mock report?